src/preferences/dialog/dlgprefinterface.cpp

```diff
@@ -439,7 +440,11 @@ void DlgPrefInterface::slotApply() {
 void DlgPrefInterface::loadTooltipPreferenceFromConfig() {
     const auto tooltipMode = static_cast<mixxx::TooltipsPreference>(
             m_pConfig->getValue(ConfigKey(kControlsGroup, kTooltipsKey),
+#ifdef Q_OS_IOS
```

Comment:
Is the OS the apprpriated criteria here? Shouldn't it be screen size, or screen size + touch instead? This also applies to RPi etc.

Reply:
Screen size would be debatable IMO, touch seems reasonable, but remember that this is only a default value and setting it depending on whether there's a mouse currently connected doesn't make much sense IMO (or would lead to surprising behavior with missing tooltips once the user connects one).

If we'd want to go this route, we would likely have to dynamically update this option based on screen size/connected input devices etc. Such a mechanism would be a lot more complex to implement than this fix, which is safe to apply given that all iOS devices use a touchscreen as their primary input.

We could implement a more complex system in future PRs, but I'd consider that out of scope for this small UX improvement which is trivial to replace.